test/cfg/std.c

```diff
@@ -219,14 +219,14 @@ void bufferAccessOutOfBounds(void)
     strncpy_s(a,5,"abcd",5);
     // string will be truncated, error is returned, but no buffer overflow
     strncpy_s(a,5,"abcde",6);
-    // TODO cppcheck-suppress bufferAccessOutOfBounds
+    // cppcheck-suppress bufferAccessOutOfBounds
     strncpy_s(a,5,"a",6);
```

Comment:
hmm I am not sure about this. It only writes 2 bytes to the buffer `a` right? A 'a' and a zero.

Reply:
Seems like the configuration was not correct.